src/styles/base/button.scss

```diff
@@ -1,35 +1,32 @@
 .button {
-    width: 185px;
-    height: 40px;
+    min-width: 120px;
```

Comment:
I strongly advise you to read about CSS properties sorting and choose one of the options.

**You can sort CSS properties:**
1. Alphabetically
2. Grouped (position / appearance)
3. Randomly (what you do now)